pages/agent/v3/hooks.md.erb

```diff
@@ -145,6 +145,11 @@ agent’s default shell. This has a few implications:
 * `$0` contains the location of the copy of the script that is running from `$TMPDIR`
 * the shebang line of the hook script has no effect
 
+<section class="Docs__troubleshooting-note">
+  <p class="Docs__note__heading">"Permission denied" error when trying to execute hooks</p>
+  <p>If your hooks don't execute and throw a <code>Permission denied</code> error, it means that they were copied to <code>/tmp</code> that is non-executable on that particular system. The directory the hooks are copied to before execution can be configured through <code>$TMPDIR</code> in the Buildkite agent's environment.</p>
```

Comment:
```suggestion
  <p>If your hooks don't execute, and throw a <code>Permission denied</code> error, it might mean that they were copied to a temporary directory on the agent that isn't executable. Configure the directory that hooks are copied to before execution using the <code>$TMPDIR</code> environment variable on the Buildkite agent, or make sure the existing directory is marked as executable..</p>
```

Reply:
I've rewritten this to be both more direct, and more vague. :-p